WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -12,11 +18,12 @@ class RegisterDomainSuggestionsViewController: UIViewController {
 
     private var site: Blog!
     var domainPurchasedCallback: ((String) -> Void)!
+    var domainAddedToCartCallback: (() -> Void)?
 
     private var domain: FullyQuotedDomainSuggestion?
     private var siteName: String?
     private var domainsTableViewController: DomainSuggestionsTableViewController?
-    private var domainType: DomainType = .registered
+    private var domainSelectionType: DomainSelectionType = .purchaseSeparately
```

Comment:
I think `.registered` corresponds to `.registerWithPaidPlan` (which happens when the user buys a plan and has domain credits they haven't used yet). 
I think `.purchaseSeparately` corresponds to `.siteRedirect` (when the user buys a domain without a plan).

Reply:
Thanks for spotting. I claimed that this naming made it *easier* but I still confused myself. 🤦‍♂️